src/openforms/js/components/form/coSign.js

```diff
@@ -28,9 +28,7 @@ const EDIT_FORM_TABS = [
                         type: 'select',
                         key: 'authPlugin',
                         label: 'Authentication method',
-                        description: `
-                            Which authentication method the co-signer must use. Note that this must be an
-                            authentication method available on the form.`,
+                        description: `Which authentication method the co-signer must use. Note that this must be an authentication method available on the form.`,
```

Comment:
```suggestion
                        description: `Which authentication method the co-signer must use. Note that this must be an authentication method available on the form.',
```
backticks are used for interpolation or multi-line strings

Reply:
Neem aan dat die eerste dan ook een normale single quote moet zijn.